devops/Makefile

```diff
@@ -52,7 +52,7 @@ ifeq ($(SRC_DIR_NAME),sovtoken)
 # remove that once sovtoken starts using indy-plenum from stable:
 # 	pypi: indy-plenum
 # 	apt: indy-plenum (stable component)
-FPM_P_DEPENDS := indy-plenum(>=1.6.53)
+FPM_P_DEPENDS := indy-node-dev(>=1.6.858)
```

Comment:
Please also have a look at `indy_plenum_ver` variable in Docker files. I think this also needs to be fixed and changed to indy-node.